### ISSUE TITLE
[Question] Module Keybindings

### ISSUE THREAD

**Issue by blackryu** [ultrabug/py3status#588]
I was wondering if it possible to set custom keybindings for Modules, in particulur pomodoro.py.

I have searched, but not found any info on this subject.
My intentions are, in the case of the pomodoro, is to set 2 keybindings (start and stop [maybe 3, restart]) but in the source code all I can see is that (sorry i have no experience with python) an event handler only reacts to mousebutton presses...

is there any way to do this? or am I missing something?
Cheers

**Comment by tobes** [comment]
Unfortunately this is not currently possible.  However it would be possible.  I have a branch that allows for keyboard control of py3status.  It does not work quite in the way you describe but it would be simple enough to adapt it to do what you are after.

The problem is it is waiting for some other PRs to be accepted and that it adds a new dependency to work.  It also need to be updated.  It is also waiting for a bug fix in i3 itself but that is for  cosmetic issue.

Basically we need to add message receiving capabilities to py3status for this.

**Comment by tobes** [comment]
@blackryu 

Hi, It is now possible to control modules via keybindings in py3status (git master).  You can do this by sending mouse click events to a module.  So for instance with pomodoro you could add this to your i3 config.

```
bindsym mod+p  exec "py3-cmd click 1 pomodoro"
```

for more on `py3-cmd` see https://py3status.readthedocs.io/en/latest/py3-cmd.html
